src/components/symptom-shark/container/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as SymptomSharkLogEntryList } from "./LogEntryList"
+export { default as SymptomSharkLogToday } from "./LogToday"
+export { default as SymptomSharkLogEntryEdit } from "./LogEntryEdit"
+export { default as SymptomSharkVisualizationCoordinator } from "./VisualizationCoordinator"
```

Comment:
Do we want to start following this pattern of renaming/prefixing when we have specific use component sub-folders?

Reply:
I think so, since I don't think there's a good way to like..."namespace" exports.  However you can always rename things on import to create a shorthand so it's not so verbose, e.g.

```
import { SymptomSharkVisualizationCoordinator as VisualizationCoordinator }
```

 But honestly open to other suggestions here, I did some googling and didn't find a perfect approach or anything.

Reply:
I think it makes sense.